src/app/pages/logs/list/logs-list-table-data-source.ts

```diff
@@ -40,13 +43,31 @@ export class LogsListTableDataSource extends TableDataSource<Log> {
     private authorizationService: AuthorizationService,
     private router: Router,
     private centralServerNotificationService: CentralServerNotificationService,
+    private activatedRoute: ActivatedRoute,
     private centralServerService: CentralServerService,
     private datePipe: AppDatePipe) {
     super(spinnerService, translateService);
     // Init
     this.initDataSource();
+    this.initFilters();
   }
 
+  public initFilters() {
+    if (this.activatedRoute && this.activatedRoute.params) {
+      this.activatedRoute.queryParams.subscribe(params => {
+        if (params['fromChargingProfile']) {
```

Comment:
Remove fromChargingProfile props in the UI and pass all the actions separated by '|' from the charging profiles pop-up.
You can also use the WindowService in window.service.ts file.